src/components/Reports/FourteenMonthReports/FourteenMonthReport.tsx

```diff
@@ -117,7 +119,7 @@ export const FourteenMonthReport: React.FC<Props> = ({
       ) : error ? (
         <Notification type="error" message={error.toString()} />
       ) : currencyTables.length > 0 ? (
-        <Box display="flex" flexDirection="column" gap={4}>
+        <Box display="flex" flexDirection="column" gap={isPrint ? 1 : 4}>
```

Comment:
All this looks great! I wonder if you could also make the main header "Contributions By Partner Currency" have less margin/padding so it doesn't take up so much space.